examples/timestamping/backend/tests/config.rs

```diff
@@ -75,21 +73,24 @@ fn init_testkit(second_time_service: bool) -> (TestKit, MockTimeProvider) {
     (testkit, mock_provider)
 }
 
-/// Creates block with `ConfigPropose` tx and returns current service configuration.
-fn propose_configuration(testkit: &mut TestKit, config: Config) -> Config {
+/// Creates block with `ConfigPropose` tx and returns `Result` with new configuration or corresponding `ExecutionError`.
```

Comment:
Micro-nit: This comment looks overly long. Also, does it make sense to *return* a `Config` if it's guaranteed to equal the input one? It could make more sense to return `Result<(), ExecutionError>` and possibly to check that the config is as expected *within* this method.